src/test/java/bio/terra/service/resourcemanagement/google/BucketResourceTest.java

```diff
@@ -127,22 +131,45 @@ public void createAndDeleteBucketTest() throws Exception {
         checkBucketDeleted(bucketResource.getName(), bucketResource.getResourceId());
     }
 
+    @Test
+    // create buckets in different regions and see if they're actually created there.
+    public void createBucketsInDifferentRegionsTest() throws Exception {
+        for (GoogleRegion region : Arrays.asList(GoogleRegion.US_CENTRAL1, GoogleRegion.US_EAST1)) {
```

Comment:
Prefer `List.of()` over `Arrays.asList()` if you have an array literal.
```suggestion
        for (GoogleRegion region : List.of(GoogleRegion.US_CENTRAL1, GoogleRegion.US_EAST1)) {
```

Reply:
Oh cool this is a Java 9+ feature. I'm so used to `Arrays.asList` for inline list creation 😅

Reply:
I'm working on retraining myself too 😃